templates/webhook.init.erb

```diff
@@ -30,7 +30,7 @@ export PATH
 start() {
     echo
     echo -n $"Starting webhook: "
-    daemon --user ${DAEMON_USER:?} <%= scope.lookupvar('r10k::params::start_pidfile_args') %> $webhook
+    start-stop-daemon --start --user ${DAEMON_USER:?} <%= scope.lookupvar('r10k::params::start_pidfile_args') %> --startas=$webhook
```

Comment:
Most of our enterprise customers use RHEL and so this will need to be conditional , likely on something like `osfamily`, There have been other PRs for ubuntu on this hook so technically I think this change is not strictly required no?